tools/lint-all

```diff
@@ -379,6 +379,7 @@ def build_custom_checkers(by_lang):
              ('zerver/views/user_settings.py', 'return json_error(error or skipped)'),
              # We can't do anything about this.
              ('zerver/views/realm_filters.py', 'return json_error(e.messages[0], data={"errors": dict(e)})'),
+             ('zerver/views/custom_profile_fields.py', 'return json_error(e.messages[0], data={"errors": dict(e)})'),
```

Comment:
Opened https://github.com/zulip/zulip/pull/3859 for this; I think that should provide the same value we're getting here, without as much linter update overhead.